test/dockerized/run.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+cd "$(dirname "$0")"
+
+if [[ -z $(docker container ls | grep cluster_scylla) ]]; then
+    echo "Scylla container must be running to execute tests (use 'make up')."
+    exit 1
+fi
+
+if ! [[ $(docker image ls -q scylla_rust_driver_testing) ]]; then
+    docker build -t scylla_rust_driver_testing .
+fi
+
+if [ -z "$CARGO_HOME" ]; then
+    CARGO_REGISTRY=$HOME/.cargo/registry
+else
+    CARGO_REGISTRY=$CARGO_HOME/registry
+fi
+
+docker run --name "scylla-rust-driver-testing" \
+    --network scylla_rust_driver_public \
+    -v "$(pwd)/../..:/scylla_driver:Z" \
```

Comment:
This has a not-nice consequence of creating a `.cargo_home` directory in the main directory which is owned by the root user. Can we avoid it somehow?